org.eclipse.scout.rt.app/src/main/java/org/eclipse/scout/rt/app/ApplicationProperties.java

```diff
@@ -358,4 +360,157 @@ public String description() {
+
+    @Override
+    public String getKey() {
+      return "scout.app.gzip.excluded.methods";
+    }
+
+    @Override
+    public String description() {
+      return "Excluded methods for " + GzipHandler.class.getName() + " (respected only if at least one is set otherwise Jetty default is used).";
+    }
+  }
+
+  public static class ScoutApplicationGzipExcludedMimeTypes extends AbstractStringListConfigProperty {
+
+    @Override
+    public String getKey() {
+      return "scout.app.gzip.excluded.mimeTypes";
+    }
+
+    @Override
+    public String description() {
+      return "Excluded MIME types for " + GzipHandler.class.getName() + " (respected only if at least one is set otherwise Jetty default is used).";
+    }
+  }
+
+  public static class ScoutApplicationGzipExcludedPaths extends AbstractStringListConfigProperty {
+
+    @Override
+    public String getKey() {
+      return "scout.app.gzip.excluded.paths";
+    }
+
+    @Override
+    public String description() {
+      return "Excluded paths for " + GzipHandler.class.getName() + " (respected only if at least one is set otherwise Jetty default is used).";
+    }
+  }
+
+  public static class ScoutApplicationGzipIncludedInflatePaths extends AbstractStringListConfigProperty {
+
+    @Override
+    public String getKey() {
+      return "scout.app.gzip.included.inflate.paths";
+    }
+
+    @Override
+    public String description() {
+      return "Included inflate paths for " + GzipHandler.class.getName() + " (respected only if at least one is set otherwise Jetty default is used).";
+    }
+  }
+
+  public static class ScoutApplicationGzipIncludedMethods extends AbstractStringListConfigProperty {
+
+    @Override
+    public String getKey() {
+      return "scout.app.gzip.included.methods";
+    }
+
+    @Override
+    public String description() {
+      return "Included methods for " + GzipHandler.class.getName() + " (respected only if at least one is set otherwise Jetty default is used).";
+    }
+  }
+
+  public static class ScoutApplicationGzipIncludedMimeTypes extends AbstractStringListConfigProperty {
+
+    @Override
+    public String getKey() {
+      return "scout.app.gzip.included.mimeTypes";
+    }
+
+    @Override
+    public String description() {
+      return "Included MIME types for " + GzipHandler.class.getName() + " (respected only if at least one is set otherwise Jetty default is used).";
+    }
+  }
+
+  public static class ScoutApplicationGzipIncludedPaths extends AbstractStringListConfigProperty {
+
+    @Override
+    public String getKey() {
+      return "scout.app.gzip.included.paths";
+    }
+
+    @Override
+    public String description() {
+      return "Included paths for " + GzipHandler.class.getName() + " (respected only if at least one is set otherwise Jetty default is used).";
+    }
+  }
+
+  public static class ScoutApplicationGzipMinSize extends AbstractPositiveIntegerConfigProperty {
```

Comment:
Do we need a default value here? The former implementation used 256 bytes as default here.
GzipHandler seems to use 32 bytes as default.

Reply:
We easily could add a default value. However I am unsure whether 256 is a perfect default value - why do we not want to gzip responses smaller? Jetty developers seem to have invested some thoughts here, they even care about a break even gzip size. I would keep the lower value on purpose. Maybe we could add additional migration notes here (previous minimum size can easily be set again by config property).

Reply:
Solved in https://github.com/eclipse-scout/scout.rt/compare/3e80c6ba06d95d99f3212f4e027768c687cc2451..fd0f424516f18d425a0a9b217cc4f4e71e021a4a